template/alpha_settings.py

```diff
@@ -10,13 +10,15 @@ class AlphaSettings(BaseAlphaSettings):
     """
     Inherit BaseAlphaSettings and assign values at least for required fields.
     """
+    # alpha name (str)
+    alpha_name = 'KDR'
     # author name (str)
```

Comment:
This line is duplicated.

Reply:
? Why duplicated ?

Reply:
Sorry, I misread it. Please close this conversation.